apps/vanilla/src/script/accessPage.ts

```diff
@@ -0,0 +1,23 @@
+import { PROFILE_PAGE } from './constants';
+import { initLoginForm, initRegisterForm } from './init';
+import { isAuthorized } from './isAuthorized';
+
+/** Check if logged in user request login page. */
+export async function isAccessableLogin(): Promise<void> {
+  const isLoggedIn = await isAuthorized();
+  if (isLoggedIn === true) {
+    location.replace(PROFILE_PAGE);
+    return;
+  }
+  initLoginForm();
+}
+
+/** Check if logged in user request login page. */
+export async function isAccessableRegister(): Promise<void> {
+  const isLoggedIn = await isAuthorized();
+  if (isLoggedIn === true) {
```

Comment:
For there is no need to use it like this for simple booleans

```suggestion
  if (isLoggedIn) {
```